### ISSUE TITLE
Vagrant - Conflicting existing scripts etc

### ISSUE THREAD

**Issue by cloudm2** [bloomberg/chef-bcpc#636]
Rebased to master and new vagrant merge caused a "perceived" issue because existing scripts such as ./vbox_create.sh and ./vbox_init exist but when called as usual caused an error because directory structure was assumed to be in some other directory. 

To be simple, we should have an install.sh script that takes care of installing via vagrant if installed or doing it the way production is done. Simple is ALWAYS better since no one reads or wants to read docs :).

**Comment by erhudy** [comment]
I did not completely excise the references to Vagrant from the old scripts. https://github.com/bloomberg/chef-bcpc/blob/master/docs/building_with_vagrant.md has the needful on how to build using the new Vagrant method. @mihalis68 and I will be working on cleaning up the bootstrap process further. I will see if I can trivially excise Vagrant references from the old scripts in the meantime.

**Comment by cloudm2** [comment]
Awesome! It's building when you follow the new instructions. Just make sure you read first...haha

**Comment by mihalis68** [comment]
I overlooked this aspect when I reviewed it. I verified that it builds according to the new method and I only thought about the non vagrant path as far as any possible breakage/unfamiliarity. Normal service will resume shortly :-)

Sent from Rivendell

----- Original Message -----
From: Chris Jones <reply@reply.github.com>
To: chef-bcpc@noreply.github.com
CC: CHRIS MORGAN
At: Jun 11, 2015 10:57:10 PM


Awesome! It's building when you follow the new instructions. Just make sure you read first...haha
